### PR TITLE
upgraded cypress 11.x -> 13.x

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What type of PR is this? 
- upgraded cypress to fix [this](https://github.com/getredash/redash/issues/6885) issue
- updated db seed function which helped to remove not needed request-cookie package
- removed unneeded cypress dockerfile to speed up CI build (temporary added image to pass CI checks)
- updated dockerignore with more unused content for redash image
- removed compose profiles

- [x] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [x] E2E Tests (Cypress)
- [x] Manually
- [ ] N/A

### CODE REVIEW

**Comment by AndrewChubatiuk** [comment]
@justinclift could you please review when have time?

**Comment by justinclift** [comment]
@gaecoli All good, I'll take a look at this in a few hours. :smile:

**Comment by AndrewChubatiuk** [comment]
> @gaecoli All good, I'll take a look at this in a few hours. 😄

@justinclift please merge this PR before other ones in a repo to speed up e2e tests execution for other pipelines

**Comment by justinclift** [comment]
Sorry @AndrewChubatiuk, this will have to be a tomorrow thing instead of today.  I've run out of time and mental energy for today. :facepalm:

Unless someone else gets to it overnight of course. :smile:

**Comment by AndrewChubatiuk** [comment]
@justinclift do you have time for it today?

**Comment by justinclift** [comment]
Yeah.  I just finished most of the other bits I need to take care of today, so I'll grab some food now then go through this. :smile:

File: client/cypress/cypress.js [hunk 1]
```diff
@@ -14,31 +12,49 @@ try {
```
Comment (justinclift) [inline client/cypress/cypress.js:12]: @eradman Would you have time to look over this cypress.js file?

The changes in it are more complex than I'm able to really parse atm.

The changes in the Cypress tests below though seem workable.  And it'd be good to get the Cypress pieces updated past Cypress 12.x. :smile:

**Comment by justinclift** [comment]
@AndrewChubatiuk This mostly looks good.  I've asked @eradman to take a look at the most complex JS file (cypress.js) in there, as it's more complex than what I can currently grok.

> removed compose profiles

Will that affect people who run the Cypress tests manually themselves?  ie from the cli, prior to making a PR

**Comment by AndrewChubatiuk** [comment]
maybe makefile should be updated as well, but current flow should not be affected

**Comment by AndrewChubatiuk** [comment]
@justinclift updated makefile, so it should not affect existing user's setup

**Review by eradman — CHANGES_REQUESTED** [review]
From my perspective this change is too large. We need to break this update into smaller steps so that we are able to fully understand.

**Comment by AndrewChubatiuk** [comment]
> From my perspective this change is too large. We need to break this update into smaller steps so that we are able to fully understand.

i can help if you have any questions. most of the changes are happening in yarn.lock, cause lot's of old packages were removed after percy upgrade

**Comment by AndrewChubatiuk** [comment]
@eradman @justinclift here's a smaller PR
https://github.com/getredash/redash/pull/6931/files

**Pull request closed without merging**